src/main/java/mekanism/common/integration/multipart/MultipartTileNetworkJoiner.java

```diff
@@ -0,0 +1,83 @@
+package mekanism.common.integration.multipart;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import io.netty.buffer.ByteBuf;
+import mcmultipart.api.container.IMultipartContainer;
+import mcmultipart.api.slot.EnumFaceSlot;
+import mcmultipart.api.slot.IPartSlot;
+import mcmultipart.api.world.IMultipartBlockAccess;
+import mekanism.common.base.ITileNetwork;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.world.IBlockAccess;
+
+public class MultipartTileNetworkJoiner implements ITileNetwork
```

Comment:
It's certainly possible, but I'm not sure what the end goal would be. Performance wise, considering we're talking about a single byte, I don't think we'd gain anything (and are more likely lose a little) by first checking if the TE is inside a MP container.

If we're looking for code style then it would mean that `MultipartTileNetworkJoiner` could remove the header byte itself and I suppose a static method (or methods, one to unwrap if we don't first check) would make the code a bit more self-documenting and eliminate the "magic number" usage in the entities themselves.

In the latter case, please let me know which route you'd rather go (check + static wrap method or static wrap/unwrap methods) and I'll make that change.